kogito-editors-js/packages/import-java-classes-component/src/components/ImportJavaClasses/ImportJavaClassesWizardFieldListTable.tsx

```diff
@@ -36,12 +52,22 @@ export const ImportJavaClassesWizardFieldListTable: React.FunctionComponent<Impo
         [pairIndex]: !expanded[pairIndex],
       });
     };
-    const getJavaClassSimpleName = (className: string) => {
-      return className.split(".").pop();
+    const decorateWithRoundBrackets = (typeName: string) => {
+      return "(" + typeName + ")";
     };
-    const decorateWithRoundBrackets = (className: string) => {
-      return " (" + className + ")";
+    const isFetchable = (field: JavaField) => {
+      if (field.dmnTypeRef === DMNSimpleType.ANY) {
+        return true;
+      }
     };
+    const fetchButton = (field: JavaField) => {
```

Comment:
shouldn't this function be wrapped by a `useCallback`?

Reply:
@vpellegrino Thank you, I think in this case the button should be re-rendered only in case the state of the JavaClasses array change.